apps/cath/demo/00/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: cath
```

Comment:
Do you need all the /env/00-01 folders?

Reply:
I don't think so, can I just get started with base and then add 00/01 as and when needed?